app/src/main/java/com/keylesspalace/tusky/components/compose/dialog/CaptionDialog.kt

```diff
@@ -90,27 +105,30 @@ class CaptionDialog : DialogFragment() {
                 ) {
                     imageView.setImageDrawable(resource)
                 }
+
+                override fun onLoadFailed(errorDrawable: Drawable?) {
+                    super.onLoadFailed(errorDrawable)
+                    imageView.hide()
```

Comment:
I think instead of just assuming it is an audio file when loading fails I'd rather pass an explicit flag. The flag could also be used to display a different hint that makes more sense for audio files.

Reply:
I was wondering (and didn't quickly find) how to find the type of the media thing.

Because always putting it in Glide says that the code expects an image there - which it probably shouldn't.